proposals/0000-remove-star-kind.rst

```diff
@@ -0,0 +1,143 @@
+Remove the * kind syntax
+========================
+
+.. proposal-number::
+.. trac-ticket::
+.. implemented::
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/143>`_.
+.. sectnum::
+.. contents::
+
+The Haskell Report uses ``*`` to denote the kind of inhabited types. As we move
+towards DependentHaskell, it is increasingly painful to support this historical
+choice of name. We propose to slowly and carefully remove this syntactic oddity
+from the language.
+
+
+Motivation
+----------
+
+Before GHC 8.0, there used to be three completely separate syntactic categories:
+terms, types, and kinds. There also was a fourth layer (sometimes called
+"sorts") that classified kinds and it had only one thing in it, called ``BOX``.
+The addition of ``-XTypeInType`` brought us a great victory in the battle with
+The Enemy of All Human Endeavor and Programming in Particular, Complexity. The
```

Comment:
I ask to forgive me for this sorry attempt at humor, as one person already pointed out to me that this story is too epic for a proposal. That said, I'm of course keeping it.